### PR TITLE
Jenkins can't do docker run -t :(

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

@moxiegirl

### CODE REVIEW

**Comment by moxiegirl** [comment]
LGTM

**Comment by SvenDowideit** [comment]
@thaJeztah :)

File: Makefile [hunk 1]
```diff
@@ -96,8 +96,8 @@ htmllint-s3: build
 		htmllint
 
 
-test: buld
```
Comment (thaJeztah) [inline Makefile]: lol

**Comment by thaJeztah** [comment]
LGTM, thanks @SvenDowideit

**Pull request merged**